leshan-server-core/src/main/java/org/eclipse/leshan/server/queue/DefaultClientAwakeTimeProvider.java

```diff
@@ -17,20 +17,20 @@
 
 import org.eclipse.leshan.server.registration.Registration;
 
-public class ClientAwakeTimeInformationDefaultImpl implements ClientAwakeTimeInformation {
+public class DefaultClientAwakeTimeProvider implements ClientAwakeTimeProvider {
```

Comment:
Finally, I find `StaticClientAwakeTimeProvider` more appropriate.
If you are OK I would change it before to integrate it in master.

Reply:
Sure! :-)